util/geoJsonCenter.js

```diff
@@ -16,4 +20,4 @@ module.exports = function( geometry ){
   }
 
   return;
```

Comment:
Are we talking about javascript here or another language, how is this function type 'void'?

If we are talking `types` in js, then exporting something which is an object seems a bit odd to me.
re: `typeof null == "object"`.

Since you feel strongly about this, I am happy to change to accommodate.